tty/zynq7000-uart/zynq7000-uart.c

```diff
@@ -68,11 +75,25 @@ static const struct {
 	{ 0xe0000000, 59, pctl_amba_uart0_clk, UART0_RX, UART0_TX },
 	{ 0xe0001000, 82, pctl_amba_uart1_clk, UART1_RX, UART1_TX }
 };
+#elif defined(__CPU_ZYNQMP)
+/* TODO: on ZynqMP we can get base address and interrupts from DTB tree */
+static const struct {
+	uint32_t base;
+	unsigned int irq;
+	uint16_t clk;
+	uint16_t rst;
+	uint16_t rxPin;
+	uint16_t txPin;
+} info[UARTS_MAX_CNT] = {
+	{ 0xff000000, 53, pctl_devclock_lpd_uart0, pctl_devreset_lpd_uart0, UART0_RX, UART0_TX },
+	{ 0xff010000, 54, pctl_devclock_lpd_uart1, pctl_devreset_lpd_uart1, UART1_RX, UART1_TX }
+};
+#endif
 
 
 static struct {
 	uart_t uart;
-	uint8_t stack[_PAGE_SIZE] __attribute__((aligned(8)));
+	uint8_t stack[_PAGE_SIZE] __attribute__((aligned(16)));
```

Comment:
Not related to this change per se - kernel should align the stack passed by the user (does not hurt to pass already aligned one, of course)